### PR TITLE
fix(www): copy button now works on iOS

### PR DESCRIPTION
**Pull request by DSchau**
## Description

This fixes an issue with the copy button that `contenteditable` and `range` must be used to work on iOS Safari. _Fun_.

### CODE REVIEW

File: www/src/components/code-block/index.js [hunk 1]
```diff
@@ -93,3 +97,15 @@ export default ({
     </Highlight>
   )
 }
+
+CodeBlock.propTypes = {
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  className: PropTypes.string,
+  disableCopy: PropTypes.bool,
```
Comment (DSchau) [inline www/src/components/code-block/index.js]:
```suggestion
  copy: PropTypes.bool,
```

File: www/src/components/copy.js [hunk 2]
```diff
@@ -13,15 +13,21 @@ import {
 } from "../utils/presets"
 
 const copyToClipboard = content => {
```
Comment (wardpeet) [inline www/src/components/copy.js]: NOT a blocking comment!

no big deal for now but have you seen the async clipboard api?
https://googlechrome.github.io/samples/async-clipboard/
https://alligator.io/js/async-clipboard-api/

It has pretty good support and has less bugs than the old api:
https://www.chromestatus.com/feature/5861289330999296

We can feature detect this and if browsers don't support it we don't show it. I don't think we should support old clipboard api.

File: www/src/components/code-block/__tests__/index.js [hunk 3]
```diff
@@ -16,7 +23,7 @@ describe(`basic functionality`, () => {
       expect(queryByText(`copy`)).toBeDefined()
     })
 
-    it(`copies text to clipboard`, () => {
+    it.only(`copies text to clipboard`, () => {
```
Comment (wardpeet) [inline www/src/components/code-block/__tests__/index.js]: Should we revert only?
```suggestion
    it(`copies text to clipboard`, () => {
```

Comment (DSchau) [reply www/src/components/code-block/__tests__/index.js]: Yasss -- awesome catch, thank you!

File: www/src/components/copy.js (hunk 2 above)
Comment (DSchau) [reply www/src/components/copy.js]: Nice. I was thinking of extracting this into a utility, and this seems like the kind of "branching logic" that can be tested and extracted. I'll do this in a bit -- love this idea!

**Comment by DSchau** [comment]
@wardpeet merging this, then gonna add the async clipboard stuff next!

File: www/src/components/copy.js (hunk 2 above)
Comment (DSchau) [reply www/src/components/copy.js]: Done in #16012

**Pull request merged**